concepts/50 React Components/05 Add DevExtreme to a React Application/63 Create a Custom Bundle.md

```diff
@@ -0,0 +1,94 @@
+In certain scenarios, you may want to utilize a custom bundle rather than an automatically generated one. The example below illustrates how to implement a custom bundle with Vite.
```

Comment:
Not quite sure how to say correct
"how to implement a custom bundle"
instead 
"how to create a custom bundle"